.storybook/tailwind.config.cjs

```diff
@@ -11,6 +11,16 @@ module.exports = {
         'neutral': '#4b5563',
         'neutral-light': '#9ca3af',
         'neutral-dark': '#1f2937'
+      },
```

Comment:
Could we make this Tailwind Config a symlink to the one at the top-level of the repo? Having two separate Tailwind configs feels a bit problematic. They could easily become out of sync.

Reply:
sure thing! updated